brave/brave-core
# Edit existing sidebar item

fix https://github.com/brave/brave-browser/issues/24964


Resolves 
<img width="389" alt="Screen Shot 2022-08-30 at 2 54 06 PM" src="https://user-images.githubusercontent.com/6786187/187360249-6bcacc0f-efec-4171-b2fb-40a19379d1ba.png"> <img width="385" alt="Screen Shot 2022-08-30 at 2 54 57 PM" src="https://user-images.githubusercontent.com/6786187/187360358-9e336aab-0029-469e-b0e7-b10f336b838f.png">

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

`npm run test brave_unit_tests -- --filter=SidebarServiceTest*`

1. Launch Brave and add any two sites to sidebar
2. Check sidebar's builtin items(ex, bookmark) doesn't have `edit...` menu in context menu
3. Check newly added two items have `edit...` context menu
4. Try to modify url or title and check it's applied
5. When another item's url is used when modifying, it's not applied
